src/components/vpncard.js

```diff
@@ -307,61 +312,9 @@ export class VPNCard extends LitElement {
       margin-right: var(--default-padding);
     }
 
-    .pill {
-      width: 45px;
-      height: 24px;
-      border-radius: 30px;
-      border: none;
-      background-color: var(--color-disabled);
-      position: relative;
-      transition: background-color 0.2s ease;
-    }
-
-    .pill:hover {
-      background-color: var(--color-disabled-hover);
-    }
-
-    .pill:active {
-      background-color: var(--color-disabled-active);
-    }
-    .on .pill,
-    .connecting .pill {
-      background-color: var(--color-enabled);
-    }
-
-    .connecting .pill:hover,
-    .on .pill:hover {
-      background-color: var(--color-enabled-hover);
-    }
-
-    .connecting .pill:active,
-    .on .pill:active {
-      background-color: var(--color-enabled-active);
-    }
-
-    .pill::before {
-      content: " ";
-      background: white;
-      display: box;
-      width: 18px;
-      height: 18px;
-      border-radius: 20px;
-      position: absolute;
-      top: 3px;
-      left: 3px;
-      transition: all 0.25s;
-    }
-
-    .connecting .pill::before,
-    .on .pill::before {
-      top: 3px;
-      left: 24px;
-    }
-
     .box.connecting svg,
     .box.connecting .timer,
-    .connecting .pill,
-    .connecting footer {
+    .connecting mz-pill .connecting footer {
```

Comment:
Do we need to separate these selectors with a comma?

Reply:
Right! fixed!